app/src/main/res/values/dimens.xml

```diff
@@ -90,4 +89,40 @@
   <dimen name="drag_drop_content_group_item_margin">8dp</dimen>
   <dimen name="walkthrough_final_no_button_margin_end">8dp</dimen>
   <dimen name="walkthrough_final_yes_button_margin_start">8dp</dimen>
+
+  <dimen name="content_item_question_view_margin_end">128dp</dimen>
```

Comment:
Looks like now this file is going to large, so can we arrange them in some groups like feature way?

Example 
```XML
<!-- Question -->
<!-- Exploration -->
<!-- Input interaction -->
<!-- Fraction interaction -->
<!-- Drag Drop interaction -->
...
...
...
```

@rt4914 WDYT?

Reply:
Yes @anandwana001 I have thought to do this in the last PR I will be making o this issue. I will resolve all the conflicts and will group them also.

Reply:
@anandwana001 I agree. That would actually make this file more readable.